src/routes/blog/posts/now-on-scoop/+page.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Introducing Scoop installation for Files
+description: I'm excited to announce that Files is now available for installation via Scoop! This new integration makes it easier than ever to sideload Files and enjoy its features without any hassle.
+thumbnail: /blog-resources/now-on-scoop/Hero.png
+date: 11/10/2024
+author: Lamparter
+---
+
+<script>
+  import { InfoBar } from "fluent-svelte";
+</script>
+
+Hello Files Community!
+
+I am thrilled to share some exciting news with you. As part of the ongoing efforts to make Files more accessible, the team has introduced Scoop as an additional method to install Files on your device.
+This means you can now install Files using Scoop, a command-line installer for Windows, known well for its simplicity and efficiency. 🚀
+
+### Why Scoop?
+
+Scoop is a popular package manager for Windows that allows you to install and manage software directly from the command line. It simplifies the installation process and ensures that you always have the latest version of your favourite apps.
+By adding Scoop to the app's list of installation methods, the aim is to provide you with more flexibility and convenience.
+
+## How to Install Files via Scoop
+
+Installing Files via Scoop is straightforward. Just follow these simple steps:
+
+1. **Open PowerShell**: Launch PowerShell with administrative privileges.
+2. **Install Scoop**: If you haven't already installed Scoop, you can do so by running the following command:
+
+   ```powershell
+   Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
+   Invoke-RestMethod -Uri https://get.scoop.sh | Invoke-Expression
+   ```
+
+3. **Install Files**: Once Scoop is installed, you can install Files by running:
+
+   ```powershell
+   scoop install nonportable/files-np
+   ```
+
+And that's it! Files is installed and ready to use. 🎉
+
+### Other Installation Methods
+
+In addition to Scoop, you can still install Files using the classic installer or via Chocolatey:
```

Comment:
Instead of listing the other methods in this post, would it be better to link to the documentation page? This way users will always have the most up to date list regardless of when they read this post.

Reply:
Sure. I added this section since the post felt too short without it.